install-manuals/all-services/README.md

```diff
@@ -1,21 +1,21 @@
 # How to run all services
 
-All services are in a `docker compose` file.
-
 ## Prerequisite
 
 ### Install docker
 
-[Install on windows](https://docs.docker.com/desktop/windows/install/)
+[Install Docker](https://docs.docker.com/engine/install/)
+
+#### Docker Desktop (not recommended for production use)
 
-[Install on linux](https://docs.docker.com/desktop/linux/install/)
+[Install Docker Desktop on windows](https://docs.docker.com/desktop/windows/install/)
 
-[Install on macos](https://docs.docker.com/desktop/mac/install/)
+[Install Docker Desktop on linux](https://docs.docker.com/desktop/linux/install/)
 
-### Clone project
+[Install Docker Desktop on macos](https://docs.docker.com/desktop/mac/install/)
 
-Clone `https://github.com/Gamify-IT/prototype`
+## Run
 
-## Run project
+All services are available as docker containers. The preferred way to host is to use a `docker compose` file.
 
-Run in the directory of the cloned project `docker-compose up --build`
+We provide a template in our [run-config](https://github.com/Gamify-IT/run-config) repository. There is also a [manual](https://github.com/Gamify-IT/run-config/hosting-template#setup) how to setup the hosting environment.
```

Comment:
the link to the manual is leading to 404

Reply:
Oh sorry its the wrong url. I will fix it